app/controllers/solutions_controller.rb

```diff
@@ -1,6 +1,21 @@
 class SolutionsController < ApplicationController
   def create
-
+    question = Question.find(params[:question_id])
```

Comment:
エラー時の処理を考えると `@` を付けてしまった方がよいでしょう。

```suggestion
    @question = Question.find(params[:question_id])
```